website/pages/docs/plugins/sources/aws/configuration.md

```diff
@@ -234,3 +236,38 @@ This is used to specify one or more accounts to extract information from. Note t
 - `skip_member_accounts` ([]string)
 
   List of OU member accounts to skip. This is useful in conjunction with `organization_units` if there are accounts under the selected OUs that should be ignored.
+
+## Advanced Configuration
+
+### Skip Tables
+
+AWS has tables that may contain many resources, nested information, and AWS-provided data.  These tables may cause certain syncs to be slow due to the amount of AWS-provided data and may not be needed.  We recommend only specifying syncing from necessary tables.  If `*` is necessary for tables, Below is a reference configuration of skip tables, where certain tables are skipped.  
+
+```yaml
+kind: source
+spec:
+  # Source spec section
+  name: aws
+  path: cloudquery/aws
+  version: "VERSION_SOURCE_AWS"
+  tables: ["*"]
+  skip_tables:
+    - aws_ec2_vpc_endpoint_services 
+    - aws_cloudtrail_events
```

Comment:
Added all 4 last_accessed_details tables for user clarity.

Reply:
It might be good to show that `skip_tables` supports wildcards?

Reply:
Good point, I think that should stay at the higher level configuration page and it will benefit users more here to have the skipped tables explicitly listed.